src/msg/async/Event.h

```diff
@@ -113,7 +113,7 @@ class EventCenter {
   CephContext *cct;
   int nevent;
   // Used only to external event
-  pthread_t owner;
+  pthread_t owner = 0;
```

Comment:
Hmm, I don't know now....

On Fri, Jul 15, 2016 at 1:41 PM, Kefu Chai <notifications@github.com> wrote:

> In src/msg/async/Event.h
> <https://github.com/ceph/ceph/pull/10300#discussion_r70924114>:
>
> > @@ -113,7 +113,7 @@ class EventCenter {
> >    CephContext *cct;
> >    int nevent;
> >    // Used only to external event
> > -  pthread_t owner;
> > +  pthread_t owner = 0;
>
> so when in_thread() gets called, owner is always initialized. and how
> come we have this valgrind warning?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ceph/ceph/pull/10300/files/0c221a95d729d779af4525618d112a9d9a0242db#r70924114>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA72YVZ6cui-lEtwQ1c3pBR-aGdgRnQXks5qVx2KgaJpZM4JMpEe>
> .
>



-- 

Best Regards,

Wheat